library/src/main/kotlin/com/github/pwittchen/rxbiometric/library/RxBiometricBuilder.kt

```diff
@@ -22,9 +22,18 @@ import java.util.concurrent.Executor
 class RxBiometricBuilder {
   internal lateinit var title: String
   internal lateinit var description: String
-  internal lateinit var negativeButtonText: String
-  internal lateinit var negativeButtonListener: DialogInterface.OnClickListener
+  internal var negativeButtonText: String? = null
+  internal var negativeButtonListener: DialogInterface.OnClickListener? = null
   internal lateinit var executor: Executor
+  internal var deviceCredentialAllowed: Boolean = false
+
+  /**
+   * Caution: You cannot call deviceCredentialAllowed() and negativeButtonText() for the same instance of BiometricPrompt.
```

Comment:
I think, we can throw `IllegalStateException` when someone is trying to do that and add appropriate unit tests for that cases.

Reply:
Actually, `PromptInfo.build()` throws `IllegalArgumentException` when deviceCredentialAllowed and negativeButtonText is set. Therefore, I think we shouldn't throw other exception so that if the situation changes in future, there'd not be need to make new release. However, I can remove the comment.

Reply:
Ok, if it's handled by the BiometricPrompt library, that's fine.